### PR TITLE
Enable secure gravatar even locally for profile/show

### PR DESCRIPTION
**Pull request by javier-menendez**
Enable secure gravatar even locally for profile/show that seems to be the only one missing

### CODE REVIEW

File: app/views/profiles/show.html.erb [hunk 1]
```diff
@@ -4,7 +4,7 @@
       <div id="avatar-frame">
         <%=
           image_tag(
-            @user.gravatar_url(size: 300, secure: request.ssl?).html_safe,
+            @user.gravatar_url(size: 300, secure: true).html_safe,
```
Comment (simi) [inline app/views/profiles/show.html.erb:7]: Would it be better to migrate this to `gravatar` helper call instead? That way we can control this from one place. Also feel free to check if there is any other call to `@user.gravatar_url` and migrate that to `gravatar` helper call)

**Comment by simi** [comment]
Thanks for the PR @javier-menendez. I have left one minor comment to address if that makes sense.

File: app/views/profiles/show.html.erb (hunk 1 above)
Comment (javier-menendez) [reply app/views/profiles/show.html.erb]: @simi I pushed a version for the helper to accept options, WDYT?

**Review by simi — APPROVED** [review]

File: app/views/profiles/show.html.erb (hunk 1 above)
Comment (simi) [reply app/views/profiles/show.html.erb]: Looks great!

**Review by segiddins — APPROVED** [review]

**Pull request merged**